pkgs/development/haskell-modules/hackage-packages.nix

```diff
@@ -69835,6 +69835,22 @@ self: {
        broken = true;
      }) {};
 
+  "dependent-map_0_3_1_0" = callPackage
+    ({ mkDerivation, base, constraints-extras, containers
+     , dependent-sum
+     }:
+     mkDerivation {
+       pname = "dependent-map";
+       version = "0.3.1.0";
+       sha256 = "1ng5hl3f4zyihz0p96kqrhr0kihcnnlbs3ridfdr16aaqm5by82z";
+       libraryHaskellDepends = [
+         base constraints-extras containers dependent-sum
+       ];
+       description = "Dependent finite maps (partial dependent products)";
+       license = "unknown";
+       hydraPlatforms = stdenv.lib.platforms.none;
```

Comment:
This seems unfortunate.  Maybe something needs to be fixed upstream??

Reply:
Oh, huh. I didn‘t notice this.

Which upstream do you mean?

Reply:
I am actually not sure if this is normal for extra-packages?

Anyways I got rid of that dependency, so it doesn’t matter for this PR anymore.